docs/ios/features/feature_memory_monitoring.md

```diff
@@ -0,0 +1,11 @@
+# Feature - Memory Monitoring
+
+Measure SDK captures memory usage periodically (defaults to 2 seconds) when the app is in the foreground.
+
+## How it works
+
+Measure SDK calculates memory usage by retrieving task-level memory information using the `task_info` API with the `TASK_VM_INFO` flavor. It extracts the `phys_footprint` value, which represents the physical memory footprint of the app in kilobytes. If the footprint value is unavailable, it falls back to using `resident_size`, which represents the resident memory size allocated to the process.
```

Comment:
What are `task_info`, `TASK_VM_INFO`, `phys_footprint` and `resident_size`.

In what cases is the footprint value not available and how it impacts what is shown on the dashboard? (side question: how does someone know on the dashboard if we fall back to resident size?)

A bit more detail on the entire process of how memory usage is captured and calculated would make this more helpful.